spec/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+
+
+  describe "GET#index" do
+    let!(:user) {FactoryBot.create(:user, id: 99)}
```

Comment:
I would not hard code the id of the user here, as it could result in nondeterministic (unpredictable) failures if a user with id 99 already exists in the database. Let the database dictate what the id will be.